src/pages/SubmissionConfig.js

```diff
@@ -222,6 +222,13 @@ export const SubmissionConfig = ({ methodId }) => {
     }) : 'No previous run set data...'
   }
 
+  const updateRunSetName = () => {
+    const timestamp = new Date().toISOString().slice(0, -5) // slice off milliseconds at the end for readability.
+    if (runSetName === '') {
+      setRunSetName(`${_.kebabCase(method.name)}_${_.kebabCase(selectedRecordType)}_${timestamp}`)
```

Comment:
it's similar to "slugify" I think. it lowercases the word and replaces spaces with dashes, among other things. it seemed like it would be useful for converting a sentence-like string into something more like an identifier (e.g. `"Target Workflow 1" -> "target-workflow-1"`)

This will be the kind of thing that we'll want Sara's opinion on